app/controllers/games_controller.rb

```diff
@@ -0,0 +1,21 @@
+class GamesController < ApplicationController
+  before_action :get_quiz, only: [:show]
+
+  def index
+  end
+
+  def show
+    render :show, locals:{message: "Welcome to quiz: #{@quiz.name}"}
+  end
+
+  def access_quiz
+    quiz = Quiz.find_by(code: params[:code])
+    redirect_to quiz_path(quiz)
+  end
```

Comment:
is this method still necessary? Why not just use the `show` method and avoid the need for a redirect?